build.sbt

```diff
@@ -104,7 +105,7 @@ lazy val `support-workers` = (project in file("support-workers"))
     integrationTestSettings,
     libraryDependencies ++= commonDependencies
   ).dependsOn(`support-services`, `support-models` % "test->test;it->test;compile->compile", `support-config`, `support-internationalisation`)
-  .aggregate(`support-services`, `support-models`, `support-config`, `support-internationalisation`, `stripe-intent`)
+  .aggregate(`support-services`, `support-models`, `support-config`, `support-internationalisation`, `stripe-intent`, `support-redemptiondb`)
```

Comment:
I considered adding a new build a bit like support-logs, but it seemed excessive to have a build agent queued up for a 10 second build, together with all the associated configuration that isn't version controlled etc.
Another option would be to have a build that just calls a shell script that copies the files, then at least as little as possible is configured in TC.

Reply:
What is it that tells SBT to copy the artifacts, the `RiffRaffArchitect` plugin?

Reply:
yes, it will do so from the contents of the `riffRaffArtifactResources` sbt key
https://github.com/guardian/support-frontend/pull/2506/files#diff-90a52a2822abc32a0ffc37008d56294aR10